app/Listeners/SendFirstVoteMessage.php

```diff
@@ -27,6 +27,8 @@ public function __construct(MessageBroker $broker)
      */
     public function handle(UserCastFirstVote $event)
     {
+        $country_code_or_global = ! empty($event->user->country_code) ? $event->user->country_code : 'global';
```

Comment:
I suspect you want `isset()` rather than `empty()`:
https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/

> isset — Determine if a variable is set and is not NULL
vs
> empty — Determine whether a variable is empty